src/backend/drakula/__init__.py

```diff
@@ -88,10 +103,30 @@ def triangulate_points(points):
     for a, b in G.edges:
         lhs = airports[a]
         rhs = airports[b]
-        con = Connection(distance_km=geo_distance(lhs.lat_lon, rhs.lat_lon).km, a=a, b=b)
+        con = Connection(
+            distance_km=geo_distance(lhs.lat_lon, rhs.lat_lon).km, a=a, b=b
+        )
         connections.append(con)
 
     return AirportsResponse(airports=airports, connections=connections)
 
 
+games_db: Dict[str, dict] = {}
+
+
+@app.post("/game")
+async def create_game(game_id: str, game_data: dict):
+    if game_id in games_db:
+        raise HTTPException(status_code=409, detail="Game ID already exists.")
+    games_db[game_id] = game_data
+    return {"id": game_id, "data": game_data}
```

Comment:
We can just return the `game_data`. Since the user supplies the `game_id` they already know it.